test/senders/beanstlakd_sender_test.go

```diff
@@ -0,0 +1,106 @@
+package senders_test
+
+import (
+	"testing"
+
+	models "github.com/shoplineapp/captin/models"
+
+	"github.com/shoplineapp/captin/senders"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestBeanstalkdSender_SendEvent_Success_WithIPAndPort(t *testing.T) {
+	sender := new(senders.BeanstalkdSender)
+	result := sender.SendEvent(
+		models.IncomingEvent{
+			Control: map[string]interface{}{
+				"beanstalkd_host": "127.0.0.1:11300",
+				"queue_name":      "foo",
+			},
+		},
+		models.Destination{
+			Config: models.Configuration{},
+		},
+	)
+
+	assert.Nil(t, result)
+}
+
+func TestBeanstalkdSender_SendEvent_Success_WithDomainAndPort(t *testing.T) {
+	sender := new(senders.BeanstalkdSender)
+	result := sender.SendEvent(
+		models.IncomingEvent{
+			Control: map[string]interface{}{
+				"beanstalkd_host": "localhost:11300",
```

Comment:
also write a test case for host like
beanstalkd.somedomain.com
and
beanstalkd.somedomain.com:11300